code/service/src/main/resources/application.conf

```diff
@@ -132,8 +132,14 @@ akka {
 				# the database table name for the offset store
 				table = "read_side_offsets"
 				table = ${?COS_READ_SIDE_OFFSET_STORE_TABLE}
+				# check migration note https://github.com/akka/akka-projection/releases/tag/v1.1.0
+				# However it is recommended that new system set the env var COS_READ_SIDE_OFFSET_USE_LOWERCASE_SCHEMA to
+				# true then the schema is lowercase un-quoted table and column names
+				use-lowercase-schema = false
```

Comment:
I think we should default this to `true` and include instructions asking people to set this to false for existing applications.

Reply:
Nope default to true will break existing application man.